src/test/ui/issues/issue-3044.rs

```diff
@@ -1,7 +1,6 @@
 fn main() {
     let needlesArr: Vec<char> = vec!['a', 'f'];
     needlesArr.iter().fold(|x, y| {
-    });
-    //~^^ ERROR mismatched types
-    //~| ERROR this function takes 2 arguments but 1 argument was supplied
+    //~^ ERROR this function takes 2 arguments but 1 argument was supplied
+});
```

Comment:
Incorrect formatting
```suggestion
    });
```